Feature/static wp scripts qa (#49)
inpsyde/reusable-workflows: docs/wp-scripts.md

> # Reusable workflows – WordPress Scripts

Reviewer: The official name is `@wordpress/scripts`. Let's use this to prevent confusion.
```suggestion
# Reusable workflows – `@wordpress/scripts`
```

## Lint

> This reusable workflow contains internally support for linting "styles" (`stylelint`), "scripts" (`eslint`) and markdown files (`markdownlint`) by executing the `@wordpress/scripts` executable `wp-scripts`.

Reviewer: Let's align the structure of this sentence to the other introductions in other workflow files.
```suggestion
This workflow runs [ESLint](https://eslint.org/), [Stylelint](https://stylelint.io/),
and [markdownlint](https://github.com/DavidAnson/markdownlint) wrapped in the
the [`@wordpress/scripts`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-scripts/)
library. It does so by executing the `wp-scripts` binary in the `./node_modules/.bin/` folder.
```

**Simplest possible example:**

```yml

> name: Front-Office QA

Reviewer: ```suggestion
name: Static code analysis assets
```

> name: Front-Office QA

Reviewer: ```suggestion
name: Static code analysis assets
```
Please also update the other code snippet below.

|-----------------------|-----------------------------------------------|----------------------------------------------------------------------------------------------|
| `NPM_REGISTRY_DOMAIN` | `https://npm.pkg.github.com/`                 | Domain of the private npm registry                                                           |
| `NODE_VERSION`        | 16                                            | Node version with which the assets will be compiled                                          |
| `PACKAGE_MANAGER`     | `yarn`                                        | Package manager with which the dependencies should be installed (`npm` or `yarn`)            |
| `NODE_OPTIONS`        | `''`                                          | Space-separated list of command-line Node options                                            |
| `LINT_TOOLS`          | `'["js", "style", "md-docs"]'`                | An array of tools supported by wp-scripts lint-*. Supported are "js", "style" and "md-docs". |
| `ESLINT_ARGS`         | `'--format stylish'`                          | Set of arguments passed to wp-script lint-js.                                                |

> | `STYLELINT_ARGS`      | `'--formatter github'`                        | Set of arguments passed to wp-script lint-style.                                             |

Reviewer: Please remove the full stops (`.`) from the descriptions and reformat the table.

Author: Removed here: https://github.com/inpsyde/reusable-workflows/pull/49/commits/7e049e35e9fd08d4d8c7fa027745c2d08ba27f94
